jwst/cube_build/ifu_cube.py

```diff
@@ -1422,6 +1423,17 @@ def setup_ifucube(self, j):
                 ifucube_model.meta.wcsinfo.ctype1 = 'MRSAL4C'
                 ifucube_model.meta.wcsinfo.ctype2 = 'MRSBE4C'
 
+#TODO
+# remove any axis4 keywords left over from other steps. These keywords interfere with
+# DS9 in understanding the wavelength planes
+# This did not work - but I left it because maybe it is something similar
+#        ifucube_model.meta.wcsinfo.ctype4 = None
+#        ifucube_model.meta.wcsinfo.crval4 = None
+#        ifucube_model.meta.wcsinfo.crpix4 = None
+#        ifucube_model.meta.wcsinfo.cdelt4 = None
+#        ifucube_model.meta.wcsinfo.cunit4 = None
+
+# set WCS information
```

Comment:
You can use `del` here:
```
try:
    del ifucube_model.meta.wcsinfo.ctype4
except AttributeError:
    pass
```

Reply:
But actually setting to `None` should work.  Just tried this out.  Hmm.

Reply:
It's not an attribute, so you can't assign None to it, and you can't del it.  It's just a keyword in the file.

Reply:
Ah, I see.  None of these are in our schemas.  So they're actually all being stored in `ifucube_model._extra_fits`.  Delete them there if you want them gone, or delete all of `_extra_fits`.

Reply:
You can see what is stored in `extra_fits` via:
```
In [90]: model.extra_fits.instance                                                                                                                           
Out[90]: 
{'PRIMARY': {'header': [['FOO', 'bar', '']]},
 'SCI': {'header': [['FOO', 'bar', '']]}}
```
And delete stuff individually:
```
In [91]: model.extra_fits.SCI.header[0]                                                                                                                      
Out[90]: ['FOO', 'bar', '']

In [92]: del model.extra_fits.SCI.header[0]
```

Reply:
del ifu_cubemodel.extra.fits.SCI,header[0] did not delete the AXIS4 keywords
I am not sure if this is correct place to delete these keywords. They should not be in
the *rate files and they are. I think they need to be removed in the CALDETECTOR1 pipeline.